Dockerfile

```diff
@@ -2,12 +2,13 @@ FROM alpine
 
 LABEL maintainer Bill Wang <ozbillwang@gmail.com>
 
-RUN apk --update add git openssh && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm /var/cache/apk/*
+COPY docker-entrypoint.sh /docker-entrypoint.sh
+
+RUN apk --update --no-cache add git openssh-client && \
+    chmod +x /docker-entrypoint.sh
```

Comment:
I strongly suggest to store the permissions in git, making this step unneeded.
```suggestion
```

Reply:
Good point! I will change the permissions